### ISSUE TITLE
Support monorepo structures

### ISSUE THREAD

**Issue by targos** [Julien-R44/adonis-vscode-extension#1]
## Why this feature is required (specific use-cases will be appreciated)?

It would be nice to be able to use the extension with a monorepo structure where the AdonisJS files are in a subdirectory.
For example, I have an app where there is an AdonisJS API in `./api` and a React frontend in `./front`.

I get this kind of error:
```
Error: Command failed: cd /d "d:/Git/project" && node ace generate:manifest
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'd:\Git\project\ace'
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

```

## Have you tried any other work arounds?

Couldn't think of one.

## Are you willing to work on it with little guidance?

Maybe, but I'm short on time at the moment.

**Comment by Julien-R44** [comment]
Hey Michaël !

Thanks for the report. I'll fix it within the week

**Comment by samueldrai** [comment]
Hi @Julien-R44
Thanks for the hardwork on this very useful extension! 
Would be delighted to see a support for monorepos (as stated by @targos)
Happy to help if needed!

**Comment by thetutlage** [comment]
@targos Are you looking for true monorepos like the one created by npm workspaces, yarn workspaces or lerna. Or mainly for apps where you have the AdonisJS code inside a sub-directory?

Plus are there any pointers on how this can be solved? Maybe you saw other tools taking some approach for custom setups

**Comment by Julien-R44** [comment]
I started an implementation, and basically I followed this approach: 
- I scan the entire workspace and subfolders looking for .adonisrc.json files
- And when launching an adonis/assembler command via the VSCode Palette, if there are several adonis projects in the workspace(s), then you have to select the one you want to run the command on. 

With this, monorepos, Adonis projects in subfolders, and multi-workspaces will be supported

**Comment by targos** [comment]
I have a monorepo with npm workspaces.
I think scanning the workspace for special files is a common strategy. That's what the TailwindCSS extension does AFAIK

**Comment by Julien-R44** [comment]
We now support Adonis projects distributed in several workspaces and also in the same workspace but in subfolders (as it can be the case with a mono repo).

I will make a release at the end of the day 👍

**Comment by Julien-R44** [comment]
Released as 0.1.0 on the marketplace 🎉

Feel free to reopen the issue if there is still a problem

**Comment by targos** [comment]
I'm going to open separate issues with further feedback
